api/api-tag-overview.md

```diff
@@ -0,0 +1,24 @@
+# API Reference Tag Overview
+
+Timescale Documentation utilizes a tagging system to describe functions in the API Reference section. These tags include "Community", "Experimental", "Toolkit", and "Experimental" (for TimescaleDB Toolkit functions). Below is more information on what each tag represents.
+
+## Community <tag type="community">Community</tag>
+
+The community tag signifies that a TimescaleDB function is available under the Timescale Community Edition license, and not available under the Apache 2 Edition license. For more information, visit our [Timescale License comparison sheet][tsl-comparison].
```

Comment:
```suggestion
This tag indicates that the function is available under the Timescale Community Edition license, and are not available under the Apache 2 Edition license. For more information, visit our [Timescale License comparison sheet][tsl-comparison].
```

Where is the "Edition" coming in here? Shouldn't it just be "the Apache 2 license"?

Reply:
Matching it with the explanations here (prescribed by Mike and Prashant) https://docs.timescale.com/timescaledb/latest/timescaledb-license-comparison/